README.md

```diff
@@ -42,33 +64,35 @@ cd uk.co.vedaconsulting.mosaico
 ./bin/setup.sh -D
 ```
 
-### Usage
+## Usage
 
 If you haven't used Mosaico before, consult the the demo and tutorial materials from http://mosaico.io/index.html.
 
 To send a new mailing, simply navigate to "Mailings => New Mailing". The CiviMail-Mosaico UI should appear.
 
 Optionally, you may design reusable templates by navigating to "Mailings => Mosaico Templates".
 
-When composing a new mailing, the default layout uses a single page.  To
+When composing a new mailing, the default layout is a simple three-step wizard.  To
 change the layout, you can update the setting `mosaico_layout` to
 `bootstrap-wizard` or `bootstrap-single`, e.g.
 
 ```
 cv api setting.create mosaico_layout=bootstrap-wizard
 ```
 
-### Having issues with this extension?
+## Having issues with this extension?
 
-Please make sure you have followed installation instructions. 
+Please make sure you have followed installation instructions.
 
 Open issues on [github](https://github.com/veda-consulting/uk.co.vedaconsulting.mosaico/issues) with:
 - screenshot of failure with any possible errors in firebug or js console
 - any related logs or backtrace from civicrm
 - tell us what version of CiviCRM and extension, you using.
 - tell us the browser you are using (name and version) and test at least a second browser to tell us if this happen in both or only one (tell us the details about the second browser too).
 
-### Development: Setup.sh
+## Development
+
+#### Setup.sh
```

Comment:
Isn't setup.sh required for installation as its downloading civicrm/mosaico library? If so then i think we shouln't add it under development. What do u think?